xbmc/pictures/PictureInfoTag.cpp

```diff
@@ -1033,7 +1033,7 @@ void CPictureInfoTag::SetInfo(int info, const std::string& value)
     }
   case SLIDE_EXIF_DATE_TIME:
     {
-      strcpy(m_exifInfo.DateTime, value.c_str());
+      strncpy(m_exifInfo.DateTime, value.c_str(), sizeof(m_exifInfo.DateTime) - 1);
```

Comment:
This string will be unterminated if length is exceeded. Adding;
m_exifInfo.DateTime[sizeof(m_exifInfo.DateTime) - 1)] = '\0';
should avoid that.
See notes here: https://linux.die.net/man/3/strncpy